core/src/test/java/hudson/cli/ListJobsCommandTest.java

```diff
@@ -61,9 +62,14 @@ public void failForNonexistingName() throws Exception {
         when(jenkins.getView("NoSuchViewOrItemGroup")).thenReturn(null);
         when(jenkins.getItemByFullName("NoSuchViewOrItemGroup")).thenReturn(null);
 
-        assertThat(runWith("NoSuchViewOrItemGroup"), equalTo(-1));
+        try {
+            assertThat(runWith("NoSuchViewOrItemGroup"), equalTo(3));
```

Comment:
`fail()` expected after the call to make sure it does not complete correctly. Btw, the `assertThat` will never be called as `runWith` will throw.